aries-site/src/data/structures/components.js

```diff
@@ -821,6 +821,28 @@ export const components = [
     relatedContent: ['Page Layouts', 'Content Layouts', 'Box', 'Main'],
     tags: ['page', 'page content'],
   },
+  {
+    name: 'PageHeader',
+    available: true,
+    category: 'Layouts',
```

Comment:
I see the category as 'Layouts' here but on the deploy preview it shows as 'Component'. Is that something else?

Reply:
Layouts is just which sub-category of components it falls under.